komodor/README.md

```diff
@@ -0,0 +1,64 @@
+# Komodor - Kubernetes Troubleshooting Platform
+
+## Overview
+
+This integration allows Datadog Monitor Alerts to be available in the Komodor Kubernetes troubleshooting platform and to suggest related services based on service connections detected by Datadog.
+
+Komodor tracks changes across your entire K8s stack, analyzes their ripple effect and provides you with the context you need to troubleshoot efficiently and independently.
+
+### Empower on-call teams
+Make the knowledge and expertise that has traditionally been held by only a few, clear and visible to developers, SRE and support teams.
+
+### Understand your kubernetes
+Gain the K8s visibility you are lacking. See your deployments on a timeline with the relevant information: what changed, what code was pushed and by whom.
+
+### Track your system end-to-end
+View data from your Git, config maps, infrastruture, alerting and other tooling such as Datadog, in one centralized and easy-to-understand display, easily link to Datadog metrics with dynamic deployment links that take you directly to the matrics you need.
+
+Troubleshoot your microservices based on the most relevant context, connections and service dependencies detected in Datadog.
+
+## Setup
+
+Once logged into the Komodor platform, you can easily install our pod-based agent on each Kubernetes cluster by using a Helm chart or Kustomize - here is a [quick overview][2] of installing the agent.
+
+Once the agent is installed you can begin setting up the Datadog integrations listed below. Once setup is completed, you can enrich deployment and service metadata with dynamic links and annotations as described below.
+
+### Datadog integration
+Komodor supports three types of integrations with Datadog.
+
+#### Komodor platform integration 
+This [initial integration][3] will allow Datadog Monitor Alerts to be available in Komodor and will suggest related services based on service dependencies detected in Datadog. 
+
+#### Datadog Webhook Integration
+The Datadog [Webhook integration][4] allows Komodor to receive alerts from Datadog Monitors. You will see all alerts in the Komodor Service View.
```

Comment:
What's the difference between this integration and the integration mentioned above? Are they both sending Datadog Monitor alerts into Komodor?